pyproject.toml

```diff
@@ -65,6 +67,8 @@ release = "python tasks/release.py"
 test = "pytest tests/"
 doc = {shell = "cd docs && mkdocs serve", help = "Start the dev server for doc preview"}
 lint = "pre-commit run --all-files"
+format = {shell = "black ./pdm ./tests && isort ./pdm ./tests && flake8 ./pdm ./tests"}
+typecheck = "mypy ./pdm ./tests"
```

Comment:
Actually, it fails with 1 error currently.

![ss 5](https://user-images.githubusercontent.com/41639488/112845275-a6793480-90df-11eb-81f2-83b164f38018.png)

It looks not fatal, but I want to fix in ideal way possibly by another PR.

Reply:
Not true, it turns out there are ~500 errors to fix, which is a huge amount

Reply:
It was actually blocking mypy.
After I excluded that directory -> and removed tests from mypy checking, it shows all errors correctly.

I first of all fixed "missing a type" errors in this PR.
It used to cause "missing a type" errors more than 50 functions, Now all errors of them are gone.